core/templates/dev/head/pages/learner-dashboard-page/learner-dashboard-page.directive.html

```diff
@@ -67,7 +67,7 @@ <h1 class="stat-value"><[$ctrl.subscriptionsList.length]></h1>
        ng-if="!$ctrl.noActivity">
     <div class="col-md-2 oppia-learner-dashboard-side-content"
          ng-class="{'oppia-learner-dashboard-mobile-mode': $ctrl.checkMobileView()}"
-         style="min-width: 200px;">
+         style="min-width: 200px;padding-left:20px">
```

Comment:
I don't think this is the right place to do it. oppia-learner-dashboard-side-content style sets it to 0 specifically. Might want to change it there instead.